src/common/commonutils/ConfigUtils.c

```diff
@@ -84,35 +89,44 @@ static int GetIntegerFromJsonConfig(const char* valueName, const char* jsonStrin
                 if (0 == valueToReturn)
                 {
                     valueToReturn = defaultValue;
-                    OsConfigLogInfo(log, "GetIntegerFromJsonConfig: %s value not found or 0, using default (%d)", valueName, defaultValue);
+                    if (IsFullLoggingEnabled())
+                    {
+                        OsConfigLogInfo(log, "GetIntegerFromJsonConfig: %s value not found or 0, using default (%d)", valueName, defaultValue);

```

Comment:
>OsConfigLogInfo [](http://example.com/codeflow?start=24&length=15)

nit: Is *Info* the right log level for this? Others are *Error*.

Reply:
No, because this also happens when the value is not found - and that's not an error. Notice this trace is not newly introduced now.

Reply:
Understood, thanks for the context!